client/app/recommendationListEntry.css

```diff
@@ -1,7 +1,134 @@
-/* image is 261 x 174 pix */
+a {
```

Comment:
A lot of repetition here with font-family, font-size etc. Might want to consider refactoring shared properties to a parent div